app/models/digital_object.rb

```diff
@@ -91,25 +91,16 @@ def number_of_children
     children.length
   end
 
-  def can_have_rights?
+  def can_have_children?
```

Comment:
Even though it's a really simple method, we should probably add a test for this and the other DigitalObject subclasses to ensure that they all behave as expected.

Reply:
Moving this to another ticket (HYACINTH-826)